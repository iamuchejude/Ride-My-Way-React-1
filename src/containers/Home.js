import React from 'react';

const Home = () => (
    <React.Fragment>
      <header className="top-nav">
      <div className="container">
        <div id="brand">
          <a href="./index.html">
            <div className="brand"></div>
          </a>
        </div>
        <nav>
          <a href="#" className="menu-icon"></a>
          <ul>
            <li>
              <a href="./signin.html">Sign In</a>
            </li>
            <li>
              <a href="./signup.html">
                <button>
                  Get Started
                </button>
              </a>
            </li>
          </ul>
        </nav>
      </div>
    </header>
    <section className="hero">
      <div className="container">
        <h1>Share a Ride.</h1>
        <p>
          Ride-My-Way is a carpooling application that provides drivers <br/>
          with the ability to create ride offers and passengers to join
          available ride offers.
        </p>
        <div>
          <a href="./rides.html">
            <button>Find a Ride</button>
          </a>
        </div>
      </div>
    </section>
    <section className="callout">
      <h1 className="title">
        How it works?
      </h1>
      <div className="container" id="feature">
        <div className="box">
          <div className="box-1"></div>
          <h1 className="title"> Create Ride Offer </h1>
          <p>
          A driver creates a ride offer on the application for a particular destination.
          </p>
        </div>
        <div className="box">
          <div className="box-2"></div>
          <h1 className="title"> Request Ride </h1>
          <p>
            A passenger request for a available ride offers to a destination,
            and waits for the driver to either accept or reject the offer.
          </p>
        </div>
        <div className="box">
          <div className="box-3"></div>
          <h1 className="title"> Take Ride </h1>
          <p>
            After successful request has been accepted. You can
            enjoy the ride with a new company and save money.
          </p>
        </div>
      </div>
    </section>
    <footer className="copyright">
      <div className="container">
        <h4> Â© 2018 RideMyWay inc.</h4>
      </div>
    </footer>
  </React.Fragment>
);

export default Home;
