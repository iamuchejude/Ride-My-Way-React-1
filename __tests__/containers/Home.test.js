import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Home from '../../src/containers/Home';

describe('render', () => {
  test('two plus two is four', () => {
    expect(2 + 2).toBe(4);
  });
  test('create a snapshot', () => {
    const wrapper = shallow(<Home />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
