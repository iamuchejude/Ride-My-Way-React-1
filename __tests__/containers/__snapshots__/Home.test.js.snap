// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render create a snapshot 1`] = `
<Fragment>
  <header
    className="top-nav"
  >
    <div
      className="container"
    >
      <div
        id="brand"
      >
        <a
          href="./index.html"
        >
          <div
            className="brand"
          />
        </a>
      </div>
      <nav>
        <a
          className="menu-icon"
          href="#"
        />
        <ul>
          <li>
            <a
              href="./signin.html"
            >
              Sign In
            </a>
          </li>
          <li>
            <a
              href="./signup.html"
            >
              <button>
                Get Started
              </button>
            </a>
          </li>
        </ul>
      </nav>
    </div>
  </header>
  <section
    className="hero"
  >
    <div
      className="container"
    >
      <h1>
        Share a Ride.
      </h1>
      <p>
        Ride-My-Way is a carpooling application that provides drivers 
        <br />
        with the ability to create ride offers and passengers to join available ride offers.
      </p>
      <div>
        <a
          href="./rides.html"
        >
          <button>
            Find a Ride
          </button>
        </a>
      </div>
    </div>
  </section>
  <section
    className="callout"
  >
    <h1
      className="title"
    >
      How it works?
    </h1>
    <div
      className="container"
      id="feature"
    >
      <div
        className="box"
      >
        <div
          className="box-1"
        />
        <h1
          className="title"
        >
           Create Ride Offer 
        </h1>
        <p>
          A driver creates a ride offer on the application for a particular destination.
        </p>
      </div>
      <div
        className="box"
      >
        <div
          className="box-2"
        />
        <h1
          className="title"
        >
           Request Ride 
        </h1>
        <p>
          A passenger request for a available ride offers to a destination, and waits for the driver to either accept or reject the offer.
        </p>
      </div>
      <div
        className="box"
      >
        <div
          className="box-3"
        />
        <h1
          className="title"
        >
           Take Ride 
        </h1>
        <p>
          After successful request has been accepted. You can enjoy the ride with a new company and save money.
        </p>
      </div>
    </div>
  </section>
  <footer
    className="copyright"
  >
    <div
      className="container"
    >
      <h4>
         Â© 2018 RideMyWay inc.
      </h4>
    </div>
  </footer>
</Fragment>
`;
